redis/src/cluster_async/mod.rs

```diff
@@ -1066,25 +1067,33 @@ where
 /// and obtaining a connection handle.
 pub trait Connect: Sized {
     /// Connect to a node, returning handle for command execution.
-    fn connect<'a, T>(info: T) -> RedisFuture<'a, Self>
+    fn connect<'a, T>(
```

Comment:
Anyway, I guess we can live with the breaking changes, obviously we'll want to wait until after the next minor release though.